src/main/java/nl/thehyve/ocdu/services/ValidationService.java

```diff
@@ -91,6 +93,10 @@
         PatientDataOcChecks checksRunner = new PatientDataOcChecks(metadata, bySubmission);
         errors.addAll(checksRunner.getErrors());
 
+        if(errors.size() == 0) {
+            openClinicaService.registerPatients(submitter.getUsername(), wsPwdHash, submitter.getOcEnvironment(), bySubmission);
```

Comment:
no. This is not necessary AFAIK. It is just about moving OpenClinicaService call (registering patients) from ValidationService elsewhere - e.g. to the UploadController.uploadPatientFile.